calour/io.py

```diff
@@ -286,8 +261,33 @@ def read_open_ms(data_file, sample_metadata_file=None, gnps_file=None, feature_m
     return exp
 
 
+def _read_metadata(ids, f, kwargs):
+    # load the sample/feature metadata file
```

Comment:
i think we should add the parameters/returns docstring even though it's an internal function (so we remember how to use it :)   )

Reply:
ok. will add that